Framework/API/inc/MantidAPI/ImplicitFunctionParameterParser.h

```diff
@@ -51,10 +51,9 @@ namespace API {
 /*
  * Deletion policy for unique pointers.
  */
-template <typename T> class DLLExport DeleterPolicy {
-public:
-  void operator()(T *pParser) { delete pParser; }
-};
+template <typename T> class DLLExport DeleterPolicy{public : void operator()(T *pParser){delete pParser;
+} // namespace API
+}; // namespace Mantid
```

Comment:
These are not the closing braces for the namespaces, those are on lines 76 and 77. I think something went wrong with the formatting.

Reply:
This version of clang-format seems to have an issue with namespaces versus classes. I've seen this in a few files.